### PR TITLE
Mark `swa_lrs` argument in `StochasticWeightAveraging` callback as required

### LINKED ISSUE #9453: LR Monitor does not work with SWA
## 🐛 Bug

LR Monitor does not work with SWA.

LR Monitor: [LearningRateMonitor](https://pytorch-lightning.readthedocs.io/en/latest/extensions/generated/pytorch_lightning.callbacks.LearningRateMonitor.html)

SWA: [Stochastic Weight Averaging](https://pytorch-lightning.readthedocs.io/en/latest/advanced/training_tricks.html#stochastic-weight-averaging)

### To Reproduce

https://colab.research.google.com/drive/1sksfJGj5d-_KJGVz3tM_ccOMnk9TPkys?usp=sharing

These lines in particular:
`lrmcb = pl.callbacks.LearningRateMonitor(logging_interval='step')`

`        stochastic_weight_avg=True,
        callbacks = [lrmcb]`

### Expected behavior

Not have this warning:

`/usr/local/lib/python3.7/dist-packages/pytorch_lightning/callbacks/lr_monitor.py:116: RuntimeWarning: You are using 'LearningRateMonitor' callback with models that have no learning rate schedulers. Please see documentation for 'configure_optimizers' method.
  RuntimeWarning,`

Because SWA is using a scheduler according to this: https://pytorch.org/blog/pytorch-1.6-now-includes-stochastic-weight-averaging/#how-to-use-swa-in-pytorch

And not have an empty list for the list of LR when inspecting the LearningRateMonitor callback instance:
`for v,k in vars(lrmcb).items():
    print(v,k)`
OUTPUT:
`logging_interval None
log_momentum False
lrs {'lr-SGD': []}
lr_sch_names []
log <bound method LightningModule.log of BoringModel(
  (layer): Linear(in_features=32, out_features=2, bias=True)
)>
log_dict <bound method LightningModule.log_dict of BoringModel(
  (layer): Linear(in_features=32, out_features=2, bias=True)
)>
last_momentum_values {}`

The line showing LR is an empty list:
`lrs {'lr-SGD': []}`

### Environment

* CUDA:
	- GPU:
	- available:         False
	- version:           10.2
* Packages:
	- numpy:             1.19.5
	- pyTorch_debug:     False
	- pyTorch_version:   1.9.0+cu102
	- pytorch-lightning: 1.4.6
	- tqdm:              4.62.0
* System:
	- OS:                Linux
	- architecture:
		- 64bit
		- 
	- processor:         x86_64
	- python:            3.7.11
	- version:           #1 SMP Sat Jun 5 09:50:34 PDT 2021

### Additional context

cc @tchaton @rohitgr7 @akihironitta

### LINKED ISSUE #11822: Make `swa_lrs` as required inside `SWACallback`
## Proposed Enhancement

Currently when swa_lrs is not set here:
https://github.com/PyTorchLightning/pytorch-lightning/blob/e3820da28a0cd0982dd1c65d7da1a0e2180454c1/pytorch_lightning/callbacks/stochastic_weight_avg.py#L34-L38

we initialize it to the optimizer lrs here:
https://github.com/PyTorchLightning/pytorch-lightning/blob/e3820da28a0cd0982dd1c65d7da1a0e2180454c1/pytorch_lightning/callbacks/stochastic_weight_avg.py#L167-L168

but during `SWALR` scheduler update, the values won't be updated because `alpha` here will be canceled out.

https://github.com/pytorch/pytorch/blob/bf233aa049c4b479fd6cb19f9b8672bb2d42b0e2/torch/optim/swa_utils.py#L281-L286

### Motivation

If we keep it as it is, it will lead to issues like this: https://github.com/PyTorchLightning/pytorch-lightning/issues/9453

### Pitch

Make `swa_lrs` in the callback as required since it's a required parameter in SWALR too and don't initialize it with any default.
https://github.com/pytorch/pytorch/blob/bf233aa049c4b479fd6cb19f9b8672bb2d42b0e2/torch/optim/swa_utils.py#L231

### Additional context

______________________________________________________________________

#### If you enjoy Lightning, check out our other projects! ⚡

- [**Metrics**](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.

- [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html): enables pure PyTorch users to scale their existing code on any kind of device while retaining full control over their own loops and optimization logic.

- [**Flash**](https://github.com/PyTorchLightning/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, fine-tuning, and solving problems with deep learning.

- [**Bolts**](https://github.com/PyTorchLightning/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks, and more for research and production with PyTorch Lightning and PyTorch.

- [**Lightning Transformers**](https://github.com/PyTorchLightning/lightning-transformers): Flexible interface for high-performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.

cc @borda @justusschock @awaelchli @akihironitta @rohitgr7 @carmocca

### PR DESCRIPTION
**Pull request by rohitgr7**
## What does this PR do?

Fixes #11822
Fixes #9453

### Does your PR introduce any breaking changes? If yes, please list them.

- The code will break for users who weren't providing value for `swa_lrs` argument.

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @carmocca

### CODE REVIEW

File: docs/source/advanced/training_tricks.rst [hunk 1]
```diff
@@ -67,7 +67,7 @@ read `this post <https://pytorch.org/blog/pytorch-1.6-now-includes-stochastic-we
 .. testcode::
 
     # Enable Stochastic Weight Averaging using the callback
-    trainer = Trainer(callbacks=[StochasticWeightAveraging(...)])
+    trainer = Trainer(callbacks=[StochasticWeightAveraging(swa_lrs=1e-2, ...)])
```
Comment (rohitgr7) [inline docs/source/advanced/training_tricks.rst:70]:
```suggestion
    trainer = Trainer(callbacks=[StochasticWeightAveraging(swa_lrs=1e-2)])
```

**Review by awaelchli — APPROVED** [review]
